chain.go

```diff
@@ -193,6 +193,62 @@ func (cc *Conn) ListChains() ([]*Chain, error) {
 	return cc.ListChainsOfTableFamily(TableFamilyUnspecified)
 }
 
+func (cc *Conn) ListChain(table *Table, chain string) (*Chain, error) {
+	conn, closer, err := cc.netlinkConn()
+	if err != nil {
+		return nil, err
+	}
+	defer func() { _ = closer() }()
+
+	tableAttr := []netlink.Attribute{
+		{Type: unix.NFTA_TABLE_NAME, Data: []byte(table.Name + "\x00")},
+	}
+	flags := netlink.Request
+	msgs := []netlink.Message{
+		{
+			Header: netlink.Header{
+				Type:  netlink.HeaderType((unix.NFNL_SUBSYS_NFTABLES << 8) | unix.NFT_MSG_GETTABLE),
+				Flags: flags,
+			},
+			Data: append(extraHeader(uint8(table.Family), 0), cc.marshalAttr(tableAttr)...),
+		},
+		{
+			Header: netlink.Header{
+				Type:  netlink.HeaderType((unix.NFNL_SUBSYS_NFTABLES << 8) | unix.NFT_MSG_GETCHAIN),
+				Flags: flags | netlink.Acknowledge,
+			},
+			Data: append(extraHeader(uint8(table.Family), 0), cc.marshalAttr(append(tableAttr, netlink.Attribute{Type: unix.NFTA_CHAIN_NAME, Data: []byte(chain + "\x00")}))...),
```

Comment:
Assign the cc.marshalAttr() to a helper variable, this line is getting rather long…

Reply:
Done.